### DISCUSSION TITLE
How to start a service from the shortcut

### DISCUSSION THREAD

**Discussion by crimera** [termux/termux-app#3875] [Q&A] [answered]
Hello, I want to start the Apache service from the shortcut with the command `sv up httpd` but it gives me the error `fail: httpd: unable to change to service directory: file does not exist` how do I do this?

**Comment by crimera** [comment, marked as answer]
nvm figured it out just needed to export the SVDIR directory

`export SVDIR=/data/data/com.termux/files/usr/var/service`

**Comment by Biswa96** [comment]
This seems to be an issue. The `sv` program should have TERMUX_PREFIX in the SVDIR variable by-default. See the SVDIR variable here https://git.sr.ht/~grimler/runit/tree/master/item/src/runsvchdir.c
